data/decks/arakune.json

```diff
@@ -0,0 +1,227 @@
+{
+	"id": "arakune",
+	"season": 5,
+	"display_name": "Arakune",
+	"cardback": "cardback_arakune.jpg",
+	"exceed_cost": 8,
+	"exceed_cost_reduced_by": "overdrive_count",
+	"has_sealed_area": true,
+	"sealed_area_is_facedown": true,
+	"exceed_to_overdrive": true,
+	"always_show_overdrive": true,
+	"on_exceed": {
+		"effect_type": "draw",
+		"amount": 2,
+		"and": {
+			"effect_type": "strike"
+		}
+	},
+	"overdrive_effect": {
+		"effect_type": "choice",
+		"add_topdeck_card_name_to_choices": [0],
+		"choice": [
+			{ "effect_type": "seal_topdeck" },
+			{ "effect_type": "pass" }
+		]
+	},
+	"ability_effects": [
+		{
+			"timing": "on_reshuffle",
+			"condition": "manual_reshuffle",
+			"effect_type": "add_set_aside_card_to_deck",
+			"id": "arakune_ntoinfinity"
+		},
+		{
+			"timing": "hit",
+			"condition": "is_special_or_ultra_attack",
+			"effect_type": "pass",
+			"and": {
+				"condition": "min_cards_in_discard",
+				"condition_amount": 1,
+				"add_topdiscard_card_name_to_choices": [0],
+				"effect_type": "choice",
+				"choice": [
+					{ "effect_type": "add_top_discard_to_overdrive" },
+					{ "effect_type": "pass" }
+				]
+			}
+		}
+	],
+	"exceed_ability_effects": [
+		{
+			"timing": "on_reshuffle",
+			"condition": "manual_reshuffle",
+			"effect_type": "add_set_aside_card_to_deck",
+			"id": "arakune_ntoinfinity"
+		},
+		{
+			"timing": "on_strike_reveal",
+			"effect_type": "choose_discard",
+			"source": "overdrive",
+			"limitation": "",
+			"destination": "discard",
+			"amount": 1,
+			"amount_min": 1,
+			"discard_effect": {
+				"condition": "discarded_matches_attack_speed",
+				"effect_type": "rangeup",
+				"amount": -8,
+				"amount2": 0,
+				"and": {
+					"effect_type": "powerup",
+					"amount": 2,
+					"and": {
+						"effect_type": "speedup",
+						"amount": 1,
+						"and": {
+							"NOTE": "Overdrive Effect",
```

Comment:
oh I did not know that overdrive effects went off any time you discarded from overdrive

Reply:
arakune explicitly calls this on his card